src/miner.cpp

```diff
@@ -101,6 +101,9 @@ CBlockTemplate* CreateNewBlock(const CScript& scriptPubKeyIn)
         return NULL;
     CBlock *pblock = &pblocktemplate->block; // pointer for convenience
 
+    /* Initialise the block version.  */
+    pblock->nVersion = CBlockHeader::CURRENT_VERSION;
```

Comment:
It's now in https://github.com/dogecoin/dogecoin/blob/1.10-dev/src/primitives/pureheader.h#L64 via https://github.com/dogecoin/dogecoin/blob/1.10-dev/src/miner.cpp#L106